requirements.txt

```diff
@@ -1,3 +1,4 @@
 ipaddress
+maxminddb-geolite2
 pytz
-scapy>=2.4.0
+scapy>=2.4.0
```

Comment:
Might consider adding `pandas` here or building out a separate requirements file for the analysis script.